W-11020951-mfa (#202)
mulesoft/docs-access-management: modules/ROOT/pages/multi-factor-authentication.adoc

* Security key
* Salesforce Authenticator

> Organizations created after April 30, 2022 require multi-factor authentication by default for all users. Organization administrators can xref:multi-factor-authentication.adoc#exempt-mfa-users[add individual user and service accounts to an exemption list] or xref:multi-factor-authentication.adoc#disable-org-mfa[disable MFA across the organization] (not recommended). Note that https://help.salesforce.com/apex/HTViewSolution?urlname=Salesforce-Multi-Factor-Authentication-FAQ[Salesforce users must have MFA enabled].

Reviewer: I don't this option "disable MFA across the organization".
I'm not sure if "Salesforce users must have MFA enabled" is relevant to anypoint.

> Organizations created after April 30, 2022 require multi-factor authentication by default for all users. Organization administrators can xref:multi-factor-authentication.adoc#exempt-mfa-users[add individual user and service accounts to an exemption list] or xref:multi-factor-authentication.adoc#disable-org-mfa[disable MFA across the organization] (not recommended). Note that https://help.salesforce.com/apex/HTViewSolution?urlname=Salesforce-Multi-Factor-Authentication-FAQ[Salesforce users must have MFA enabled].

Author: ```suggestion
Organizations created after April 30, 2022 require multi-factor authentication by default for all users. Organization administrators can xref:multi-factor-authentication.adoc#exempt-mfa-users[add individual user and service accounts to an exemption list]. Note that https://help.salesforce.com/apex/HTViewSolution?urlname=Salesforce-Multi-Factor-Authentication-FAQ[Salesforce users must have MFA enabled].
```

If your organization requires multi-factor authentication and you have removed all verification methods, you must configure at least one verification method before you save your changes.
. Click *Save*.

[[disable-org-mfa]]
== Disable Multi-Factor Authentication Requirement for All Organization Users

> When you disable the multi-factor authentication requirement, existing users no longer use it to log in to Anypoint Platform and new users log in without being prompted to configure it. Note that disabling multi-factor authentication requirements does not automatically remove multi-factor authentication from user accounts. Individual users who have independently enabled multi-factor authentication must manually remove it in their profile settings.

Reviewer: I think it is outdated. You cannot disable MFA. You can make it either optional or mandatory

> When you disable the multi-factor authentication requirement, existing users no longer use it to log in to Anypoint Platform and new users log in without being prompted to configure it. Note that disabling multi-factor authentication requirements does not automatically remove multi-factor authentication from user accounts. Individual users who have independently enabled multi-factor authentication must manually remove it in their profile settings.

Author: ```suggestion
NOTE:
Organizations created after April 30, 2022 cannot disable the multi-factor authentication requirement in their organizations and must add individual accounts to the exemption list to bypass the requirement.

When you disable the multi-factor authentication requirement, existing users no longer use it to log in to Anypoint Platform and new users log in without being prompted to configure it. Note that disabling multi-factor authentication requirements does not automatically remove multi-factor authentication from user accounts. Individual users who have independently enabled multi-factor authentication must manually remove it in their profile settings.
```

If your organization requires multi-factor authentication and you have removed all verification methods, you must configure at least one verification method before you save your changes.
. Click *Save*.

[[disable-org-mfa]]
== Disable Multi-Factor Authentication Requirement for All Organization Users

[NOTE]
Organizations created after April 30, 2022 cannot disable the multi-factor authentication requirement in their organizations and must add individual accounts to the exemption list to bypass the requirement.

> When you disable the multi-factor authentication requirement, existing users no longer use it to log in to Anypoint Platform and new users log in without being prompted to configure it. Note that disabling multi-factor authentication requirements does not automatically remove multi-factor authentication from user accounts. Individual users who have independently enabled multi-factor authentication must manually remove it in their profile settings.

Author: ```suggestion
When you disable the multi-factor authentication (MFA) requirement, new users who log in to the organization do not receive a prompt to configure MFA. Note that disabling MFA does not automatically remove MFA from user accounts. Individual users who have configured multi-factor authentication must manually remove it in their profile settings.
```
